scripts/dev-bundle-tool-package.js

```diff
@@ -15,7 +15,7 @@ var packageJson = {
     "node-gyp": "8.0.0",
     "node-pre-gyp": "0.15.0",
     typescript: "4.5.4",
```

Comment:
@Grubba27 that is a right place to update typescript version, not in eslint-plugin...

Reply:
Yeah, I had in mind that needed only to update  the meteor-babel and meteor-babel package but I found this [commit](https://github.com/meteor/meteor/commit/f00f7534a2daa430ee5b63eac427878bba694310#diff-9aa09d29eebfdafcd71c7d703578e34f6558f0aa11ad7f15a23b44fc1a05492d)
Will add this one to the internal docs in order to never be missed this again